acceptance/tests/bolt_apply.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'bolt_command_helper'
+require 'json'
+
+test_name "bolt apply should apply manifest block on remote hosts via ssh and winrm" do
+  extend Acceptance::BoltCommandHelper
+
+  ssh_nodes = select_hosts(roles: ['ssh'])
+  winrm_nodes = select_hosts(roles: ['winrm'])
+  targets = ssh_nodes + winrm_nodes
+
+  # Puppet 6 doesn't support OSX 10.11, so skip those hosts if present
+  targets -= select_hosts(platform: [/osx-10.11/])
+
+  skip_test('no applicable nodes to test on') if targets.empty?
+
+  bolt_command = "bolt apply -e 'notify { \"hello world\": }' --nodes #{targets.join(',')}"
+  flags = {
+    '--format' => 'json'
+  }
+
+  step "execute `bolt apply` with json output" do
```

Comment:
Should this also include a test around applying a manifest file?

Reply:
The integration tests may be enough.